samcli/lib/build/app_builder.py

```diff
@@ -332,7 +337,12 @@ def _stream_lambda_image_build_logs(self, build_logs: List[Dict[str, str]], func
                     self._stream_writer.flush()
 
     def _build_layer(
-        self, layer_name: str, codeuri: str, specified_workflow: str, compatible_runtimes: List[str]
+        self,
+        layer_name: str,
+        codeuri: str,
+        specified_workflow: str,
+        compatible_runtimes: List[str],
+        env_vars: Optional[Dict] = None,
```

Comment:
Are these specifically for container-build envvars? Envvars are by definition already in the environment and don't otherwise require special handling. In that case I'd make it clear it's only for containers.

Reply:
that is correct, are you suggesting a name change? or a comment would be sufficient? I would want to use env_vars name since all the other constructors use the field name env_vars